plugins/make_plugin_rpm

```diff
@@ -0,0 +1,241 @@
+#!/bin/bash
+
+##--------------------------------------------------------------------
+## Copyright (c) 2019 Dianomic Systems
+##
+## Licensed under the Apache License, Version 2.0 (the "License");
+## you may not use this file except in compliance with the License.
+## You may obtain a copy of the License at
+##
+##     http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+##--------------------------------------------------------------------
+##
+## Author: Mark Riddoch
+##
+
+set -e
+
+# Default branch to package
+branch=master
+
+usage="$(basename "$0") [-a] [-c] [-h] [-b <branch>] repository ...
+This script is used to create the RPM package for a FogLAMP plugin
+
+Arguments:
+ -h	- Display this help text
+ -c	- Remove all the old versions saved in format .XXXX
+ -a	- Remove all the versions, including the last one
+ -b	- Branch to base package on"
+
+
+while getopts ":hcab:" opt; do
+	case "$opt" in
+		c)
+			echo -n "Cleaning the package archive of older versions..."
+			find "./archive" -maxdepth 1 | grep '.*\.[0-9][0-9][0-9][0-9]' | xargs rm -rf
```

Comment:
Remove `-maxdepth 1` as we create duplicate package with number (.0001) under root i.e `/archive`